"npm run test" Fails

Hi, currently running "npm run test" returns an error.
This happened since c735618f0180198fca0f33085b3a2aa1d91421b3

How to reproduce:

`npm install`

`npm run build`

`npm run test`

result of `npm run test`:

```
> tsjs-lib-generator@ test C:\Users\Thomas\Desktop\Threejs\TSJS-lib-generator
> tsc -p ./tsconfig.json && node ./lib/index.js && node ./lib/test.js

Test failed: 'webworker.iterable.generated.d.ts' is different from baseline file.

+ expected - actual

      */
     values(): IterableIterator<string>;
 }

-interface IDBDatabase {
-    /**
-     * Returns a new transaction with the given mode ("readonly" or "readwrite") and scope which can be a single object store name or an array of names.
-     */
-    transaction(storeNames: string | Iterable<string>, mode?: IDBTransactionMode): IDBTransaction;
-}
-
 interface IDBObjectStore {
     /**
      * Creates a new index in store with the given name, keyPath and options and returns a new IDBIndex. If the keyPath and options define constraints that cannot be satisfied with the data already in store the upgrade transaction will abort with a "ConstraintError" DOMException.
      *

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! tsjs-lib-generator@ test: `tsc -p ./tsconfig.json && node ./lib/index.js && node ./lib/test.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the tsjs-lib-generator@ test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```

--- a/baselines/webworker.iterable.generated.d.ts
+++ b/baselines/webworker.iterable.generated.d.ts
@@ -50,6 +50,13 @@ interface Headers {
     values(): IterableIterator<string>;
 }
 
+interface IDBDatabase {
+    /**
+     * Returns a new transaction with the given mode ("readonly" or "readwrite") and scope which can be a single object store name or an array of names.
+     */
+    transaction(storeNames: string | Iterable<string>, mode?: IDBTransactionMode): IDBTransaction;
+}
+
 interface IDBObjectStore {
     /**
      * Creates a new index in store with the given name, keyPath and options and returns a new IDBIndex. If the keyPath and options define constraints that cannot be satisfied with the data already in store the upgrade transaction will abort with a "ConstraintError" DOMException.

